Makefile

```diff
@@ -162,7 +168,7 @@ diff-cover.html: coverage-gcovr.xml coverage.xml
 		--html-report diff-cover.html
 
 nosetests.xml: FORCE
-	./setup.py nosetests --with-xunit
+	./setup.py nosetests --with-xunit --attr ${TESTATTR}
```

Comment:
setup.cfg provides other default values and acts as a fallback if the Setuptools nosetests command is run outside of the Makefile